tracer/src/Datadog.Trace.Coverage.collector/Datadog.Trace.Coverage.collector.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
```

Comment:
Looking at the Coverlet collector you referenced, it looks like they name their assembly `coverlet.collector` so it can be found with the `*collector.dll` pattern, but the namespace in their library is `Coverlet.Collector`. What do you think of doing the same and renaming all of our namespaces to `Datadog.Trace.Coverage.Collector`?

Reply:
good catch,  I made the namespace change in [0d4f715](https://github.com/DataDog/dd-trace-dotnet/pull/2739/commits/0d4f7150158aa5182b26593c3377982f826772ec)